### PR TITLE
Oracle doc update

### PR DESCRIPTION
**Pull request by CamyDee**
• Update PowerBI Desktop connect to Oracle documentation
	• Update Gateway connect to Oracle documentation

Align both of these docs so that they agree and share common troubleshooting steps

### CODE REVIEW

**Comment by PRMerger14** [comment]
@CamyDee : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/connect-data/desktop-connect-oracle-database.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ To determine which version of Power BI Desktop is installed, select **File** > *
 - For the 64-bit version of Power BI Desktop, [download and install the 64-bit Oracle client](https://www.oracle.com/database/technologies/odac-downloads.html).
 
 > [!NOTE]
+> Ensure you choose a version of Oracle Data Access Client (ODAC) which is compatible with your Oracle Server. For instance, ODAC 12.x does not support Oracle Server version 9.
```
Comment (rufengli) [inline powerbi-docs/connect-data/desktop-connect-oracle-database.md:37]: Sorry this statement I gave to you ealier is not accurate.  It should be:

ODAC 12.x does not always work with Oracle Server version 9.

File: powerbi-docs/connect-data/desktop-connect-oracle-database.md [hunk 2]
```diff
@@ -34,6 +34,8 @@ To determine which version of Power BI Desktop is installed, select **File** > *
...
+> It is highly recommended that you choose the Windows installer of the Oracle Client, rather than the xcopy installer.
```
Comment (rufengli) [inline powerbi-docs/connect-data/desktop-connect-oracle-database.md:38]: I believe they cannot use xcopy installer or they need to do a lot of configuration change.  We don't know exactly what config they need to change if xcopy installer ever works.

**Comment by PRMerger13** [comment]
@CamyDee : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@CamyDee : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/connect-data/desktop-connect-oracle-database.md (hunk 2 above)
Comment (CamyDee) [reply powerbi-docs/connect-data/desktop-connect-oracle-database.md]: changed to "Choose the Windows Installer"

File: powerbi-docs/connect-data/desktop-connect-oracle-database.md (hunk 1 above)
Comment (CamyDee) [reply powerbi-docs/connect-data/desktop-connect-oracle-database.md]: Changed to "...does not always work..."

**Comment by davidiseminger** [comment]
#sign-off

**Pull request merged**